tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -214,8 +323,53 @@ def test_45525(self):
         LOGGER.info("STARTED: Test Delete source object when copy multipart object is in "
                     "progress in same bucket")
         self.create_put_parallel_copy_and_delete_object((self.srcbuck, "src-obj", self.srcbuck,
-                                                         "dest-obj", "NoSuchKey", 5,
+                                                         "dest-obj", errmsg.NO_SUCH_KEY_ERR, 5,
                                                          self.file_path), (self.srcbuck, "src-obj"),
                                                         is_mpu=True)
         LOGGER.info("ENDED: Test Delete source object when copy multipart object is in "
                     "progress in same bucket")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-45526")
+    @CTFailOn(error_handler)
+    def test_45526(self):
+        """Test Delete destination object when copy multipart object is in progress
+        in same bucket"""
+        LOGGER.info("STARTED: Test Delete destination object when copy multipart object is in "
+                    "progress in same bucket")
+        self.create_put_parallel_copy_and_delete_object(
```

Comment:
Let's not mix indentation styles, we can follow the format being followed at line 270 (after moving args to new lines).

Applicable to other calls to this method as well.

Reply:
Kept same indentation style in all calls to this method as followed at line 270.